nautobot/dcim/tables/devices.py

```diff
@@ -1005,8 +1051,10 @@ class DeviceModuleBayTable(ModuleBayTable):
         '"></i> <a href="{{ record.get_absolute_url }}">{{ value }}</a>',
         attrs={"td": {"class": "text-nowrap"}},
     )
+    module_family = tables.Column(linkify=True, verbose_name="Family")
     installed_module = tables.Column(linkify=True, verbose_name="Installed Module")
     installed_module__status = ColoredLabelColumn(verbose_name="Installed Module Status")
+    requires_first_party_modules = BooleanColumn(verbose_name="Requires First-Party Modules")
```

Comment:
A bit of a long column verbose_name - maybe `"1st-Party Only"` would be better?

Reply:
Agreed